plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx

```diff
@@ -20,6 +20,7 @@ import { BlockInstance } from '@wordpress/blocks';
  * Internal dependencies
  */
 import BlockPreview from './block-preview';
+import { useCallback } from 'react';
```

Comment:
Should we import from `@wordpress/element` instead?
```suggestion
import { useCallback } from '@wordpress/element';
```